### DISCUSSION TITLE
Api Key validation

### DISCUSSION THREAD

**Discussion by securigy** [betalgo/openai#91] [General]
How can API Key be validated that it is valid key? I'd rather not call API to generate text and discard it just to validate the key...
If you put in a fake key ("1234") you get back some strange error messages...
Any ideas?

**Comment by gotmike** [comment]
i think listing the models would not incur a charge, but would require a valid api key to authenticate.

like this...
https://github.com/betalgo/openai/blob/cf58906d9c8efaad19e4d7ce4f82e75c53491a90/OpenAI.Playground/TestHelpers/ModelTestHelper.cs#L7

**Reply by kayhantolga** [reply]
Should we add a validation method 🤔

**Reply by gotmike** [reply]
I think it is easy enough. It's your call though. Sometimes people prefer to mirror the API. But i think it would be useful. Especially in tests. I can make a PR for it if u want.
